daheinze-oscheer/integers/adder.go

```diff
@@ -0,0 +1,5 @@
+package integers
+
+func Add(a, b int) int {
+	return a + b
```

Comment:
Please add a comment to the Add function:

```suggestion
// Add returns the sum of two integers
func Add(a, b int) int {
	return a + b
```